.travis.yml

```diff
@@ -2,3 +2,24 @@ language: java
 jdk:
     - oraclejdk11
 
+before_install:
+     - echo "Deleting old .m2 artifacts..."
+     - rm -rf $HOME/.m2/repository/net/wasdev
+     - rm -rf $HOME/.m2/repository/io/openliberty
+     - echo 'Installing ci.common lib ....'
```

Comment:
Can we also include these commands in a .sh script at the root of the repo? Something like a `setup.sh` script perhaps. Probably don't need to include the `rm -rf` steps though

Reply:
ok, addressed